lib/Basics/CrashHandler.cpp

```diff
@@ -224,29 +205,31 @@ void crashHandler(int signal, siginfo_t* info, void*) {
       int numFrames = backtrace(traces, maxFrames);
```

Comment:
shouldn't we limit numFrames to some arbitrary number that we can write into the stacktrace with pointer sizes to print?